metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-ferm",
-  "version": "2.5.1-rc0",
+  "version": "2.5.1-rc1",
```

Comment:
You don't need to update the version. Our release process will take care of this.

Reply:
Ah sorry this would be my lack of experience,..